src/util/helpers.py

```diff
@@ -93,3 +119,26 @@ def fill_dates(df, first_row = False):
         df.iloc[1::2][['Month','Day']] = df[['Month','Day']].shift(1).iloc[1::2]
     return df
 
+# Month Dictionary
+month_dict = {
+    '01': 'Jan',
+    '02': 'Feb',
+    '03': 'Mar',
+    '04': 'Apr',
+    '05': 'May',
+    '06': 'Jun',
+    '07': 'Jul',
+    '08': 'Aug',
+    '09': 'Sep',
+    '10': 'Oct',
+    '11': 'Nov',
+    '12': 'Dec'
+    }
+
+# Converts date from 'yyyymmdd' to month and day
+def convert_integer_date(date, month_dict = month_dict):
+    new_date = datetime.strptime(date.strip(), '%Y%m%d').strftime('%m/%d/%Y')
+    first_slash = new_date.find('/')
+    second_slash = new_date.find('/', first_slash + 1)
+    return month_dict[new_date[:first_slash]], new_date[first_slash + 1: second_slash]
```

Comment:
alternatively, could use split to split `new_date` into an array of parts using "/" as the delimeter


```suggestion
new_date_parts = new_date.split('/')
return month_dict[new_date_parts[0]], new_date_parts[1]
```